docker-push/README.md

```diff
@@ -1,30 +1,29 @@
-# Docker Push to GAR Composite Action
+# docker-push
```

Comment:
If I recall correctly, this is automatically generated, so I'm going to skip reviewing it.

Reply:
At present this is not automatically generated, but I'd very much like it to be if you know of an easy way to do that!

Reply:
Oh, my bad. I'm thinking of terraform docs.